services/movie.py

```diff
@@ -0,0 +1,41 @@
+import init_django_orm  # noqa: F401
+
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if not genres_ids and not actors_ids:
+        return queryset
+    if genres_ids and not actors_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids)
+    if not genres_ids and actors_ids:
+        return Movie.objects.filter(actors__id__in=actors_ids)
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+        queryset = queryset.filter(actors__id__in=actors_ids)
+        return queryset

```

Comment:
So hard code....
You can only two cases:
1.  genres_ids is not None
2. actors_ids is not None